src/Icons/doc/index.rst

```diff
@@ -43,6 +77,61 @@ define the HTML attributes added to the ``<svg>`` element:
     {{ ux_icon('user-profile', {height: '16px', width: '16px', aria-hidden: true}) }}
     {# renders <svg height="16" width="16" aria-hidden="true"> ... </svg> #}
 
+.. code-block:: yaml
+
+    # config/packages/ux_icons.yaml
+    ux_icons:
+        iconify:
+            enabled: true
+            on_demand: true
+            endpoint: 'https://api.iconify.design'
+
+Icon Sets
+~~~~~~~~~
+
+Icon Sets many icon sets available, each with their own unique style and set of icons
+provide a wide range of icons for different purposes, while maintaining a consistent
+look and feel across your application.
+
+With more than 200,000 icons available, you can find the perfect icon for your project.
+
+Here are some of the most popular icon sets available:
+
+===================== ======= ========= ======= =======================
+Icon Set               Icons  License   Prefix       Example
```

Comment:
Not sure if it's fully supported yed, but table alignment in RST works by adding spaces before or after the text inside a column. So, to align contents (or to do that when our parser supports it in the future), this column should be:

```rst
=======
  Icons
=======
```

The word `Icons` should be right aligned with the `=======`